packages/application-components/package.json

```diff
@@ -45,21 +45,21 @@
   },
   "devDependencies": {
     "@commercetools-frontend/ui-kit": "9.4.0",
-    "@storybook/addon-actions": "5.0.10",
-    "@storybook/addon-info": "5.0.10",
-    "@storybook/addon-knobs": "5.0.10",
-    "@storybook/addon-links": "5.0.10",
-    "@storybook/addon-options": "5.0.10",
-    "@storybook/addon-storysource": "5.0.10",
-    "@storybook/react": "5.0.10",
-    "@storybook/theming": "5.0.10",
+    "@storybook/addon-actions": "5.0.11",
+    "@storybook/addon-info": "5.0.11",
+    "@storybook/addon-knobs": "5.0.11",
+    "@storybook/addon-links": "5.0.11",
+    "@storybook/addon-options": "5.0.11",
+    "@storybook/addon-storysource": "5.0.11",
+    "@storybook/react": "5.0.11",
+    "@storybook/theming": "5.0.11",
     "html-loader": "0.5.5",
     "markdown-loader": "5.0.0",
     "moment-locales-webpack-plugin": "1.0.7",
     "react": "16.8.6",
     "react-dom": "16.8.6",
     "react-intl": "2.8.0",
-    "react-testing-library": "6.1.2",
+    "react-testing-library": "7.0.0",
```

Comment:
Please update the peer dependency range as well

Reply:
d0a3a5f. We can support both. It's more of a consumer decision I would say as we support the stricter v7 interally.